qiskit/ignis/verification/randomized_benchmarking/rb_utils.py

```diff
@@ -245,3 +247,195 @@ def twoQ_clifford_error(ngates, gate_qubit, gate_err):
     alpha2Q_cliff = 1/5*(np.sum(alpha1Q)+3*np.prod(alpha1Q))*alpha2Q
 
     return (1-alpha2Q_cliff)*3/4
+
+
+def calculate_1q_epg(gate_per_cliff: Dict[int, Dict[str, float]],
+                     epc: float,
+                     qubit: int) -> Dict[str, float]:
+    r"""
+    Convert error per Clifford (EPC) into error per gates (EPGs) of single qubit basis gates.
```

Comment:
There should be a blank line after the first sentence, otherwise this is rendered as a single paragraph in the docs.